include/in_process_server.h

```diff
@@ -254,6 +254,8 @@ class Client
         std::chrono::seconds timeout = std::chrono::seconds{10});
 
     void roundtrip();
+
+    auto server() const -> wlcs::Server&;
```

Comment:
Given my comment below, I'm unconvinced that there's a good motivation for adding this.

Maybe, what is needed is a "check_interface_expected()" method?

Reply:
whether or not it's needed here, I think it's useful to have. I'll pull it out of this PR though.

Reply:
I think it likely to lead to a [train wreck](https://devcards.io/train-wreck).

Reply:
Ehh... that's fair. Perhaps Client should hold a server reference, but not expose it directly. Either way, I'll get rid of this for now.